.eslintrc.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-    extends: ['landr'],
+    extends: ['./packages/eslint-config-landr'],
```

Comment:
are you telling me that the linting was not working properly on the linting repo ? 🙄

Reply:
It was working but the thing is that it was taking the one on npm, now it uses directly the one in the repo